tests/trx_finality_status_forked_test.py

```diff
@@ -0,0 +1,332 @@
+    #an error occurs)
+    numPasses = 2
+    blocksPerProducer = 12
+    blocksPerRound = totalProducers * blocksPerProducer
+    count = blocksPerRound * numPasses
+    while nonProdNode.verifyAlive() and count > 0:
+        # wait on prodNode 0 since it will continue to advance, since defproducera and defproducerb are its producers
+        Print("Wait for next block")
+        assert prodNodes[0].waitForNextBlock(timeout=6), Print("Production node 0 should continue to advance, even after bridge node is killed")
+        count -= 1
+
+    assert not nonProdNode.verifyAlive(), Print("Bridge node should have been killed if test was functioning correctly.")
+
+    def getState(status):
+        assert status is not None, Print("ERROR: getTransactionStatus failed to return any status")
+        assert "state" in status, \
+            Print("ERROR: getTransactionStatus returned a status object that didn't have a \"state\" field. state: {}".
+                  format(json.dumps(status, indent=1)))
+        return status["state"]
+
+    transferAmount = 10
+    prodNodes[1].transferFunds(cluster.eosioAccount, account1, "{}.0000 {}".format(transferAmount, CORE_SYMBOL), "fund account")
+    transId = prodNodes[1].getLastSentTransactionId()
+    retStatus = prodNodes[1].getTransactionStatus(transId)
+    state = getState(retStatus)
+
+    localState = "LOCALLY_APPLIED"
+    inBlockState = "IN_BLOCK"
+    irreversibleState = "IRREVERSIBLE"
+    forkedOutState = "FORKED_OUT"
+    unknownState = "UNKNOWN"
+
+    assert state == localState, \
+        Print("ERROR: getTransactionStatus didn't return \"{}\" state.\n\nstatus: {}".format(localState, json.dumps(retStatus, indent=1)))
+
+    assert prodNodes[1].waitForNextBlock(), Print("Production node 1 should continue to advance, even after bridge node is killed")
+
+    # since the Bridge node is killed when this producer is producing its last block in its window, there is plenty of time for the transfer to be
+    # sent before the first block is created, but adding this to ensure it is in one of these blocks
+    numTries = 2
+    preInfo = prodNodes[1].getInfo()
+    while numTries > 0:
+        retStatus = prodNodes[1].getTransactionStatus(transId)
+        state = getState(retStatus)
+        if state == inBlockState:
+            break
+        numTries -= 1
+        assert prodNodes[1].waitForNextBlock(), Print("Production node 1 should continue to advance, even after bridge node is killed")
+
+    postInfo = prodNodes[1].getInfo()
+    Print("preInfo: {}\n\npostInfo: {}".format(json.dumps(preInfo, indent=1), json.dumps(postInfo, indent=1)))
+
+    assert state == inBlockState, \
+        Print("ERROR: getTransactionStatus didn't return \"{}\" state.\n\nstatus: {}".format(inBlockState, json.dumps(retStatus, indent=1)))
+
+    Print("Relaunching the non-producing bridge node to connect the nodes")
+    if not nonProdNode.relaunch(nonProdNode.nodeNum):
```

Comment:
Should not be passing any value to relaunch().  It relaunches the node being controlled by that instance.  It does not accept a node ID.